bigtable/client/rpc_backoff_policy.h

```diff
@@ -90,6 +91,27 @@ class ExponentialBackoffPolicy : public RPCBackoffPolicy {
   std::chrono::microseconds maximum_delay_;
 };
 
+/**
+ * Implement a simple randomized backoff policy.
```

Comment:
Please explain the comment why randomization is useful.  Not all users would be familiar with the problem.

Reply:
Done.